src/XrdXrootd/XrdXrootdFile.cc

```diff
@@ -190,6 +190,7 @@ void XrdXrootdFile::Serialize()
        syncWait = &mySem;
```

Comment:
Do we still need this line? What is happening with `syncWait` between here and the line you are adding below?

Reply:
I think we still need this line. syncWait is used by XrdXrootdFile::Ref() (from another thread) when refCount becomes zero or negative.

Reply:
Ok, thanks for the explanation!